### PR TITLE
Hotfix: rc/1.115.2

### PR DESCRIPTION
**Pull request by samrichca**
# Description of changes

- fix desktop user dropdown button
- fix submitted data feature flag (see: https://github.com/icgc-argo/platform-ui/pull/2400/files https://github.com/icgc-argo/platform-ui/pull/2420)

## Type of Change

- [x] Bug
- [ ] Dependency updates
- [ ] Feature
- [ ] Refactoring
- [x] Release candidate
- [ ] Styling
- [ ] Testing
- [ ] Other (please describe)

## Checklist before requesting review

- Design (select one):
  - [x] Matches design:
    - component sizes, spacing, and styles
    - font size, weight, colour
    - spelling has been double checked
  - [ ] No design provided, screenshot of changes included in PR
  - [ ] No changes to UI
- Back-end (select one):
  - [ ] Back-end PRs merged, tested on develop, and linked in this PR
  - [x] No back-end changes
- [x] Manually tested changes
- [x] Added copyrights to new files
- [x] Connected ticket to PR - no, hotfix

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "argo-platform-ui",
-  "version": "1.115.1",
+  "version": "1.115.2",
   "scripts": {
```
Comment (daniel-cy-lu) [inline package.json:4]: Update version number

File: components/NavBar.tsx [hunk 2]
```diff
@@ -275,8 +275,8 @@ export default function Navbar({ hideLinks = false, disableLogoLink = false }) {
             {userModel && (
               <FocusWrapper
                 onClick={(e) => {
-                  e.stopPropagation();
                   if (isMobileLayout) {
+                    e.stopPropagation();
                     setUsingProfileOptions(true);
```
Comment (daniel-cy-lu) [inline components/NavBar.tsx:280]: Placing method in 'if' bracket

**Review by daniel-cy-lu — APPROVED** [review]

File: components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx [hunk 3]
```diff
@@ -984,7 +990,7 @@ const DonorSummaryTable = ({
           },
           width: 95,
         },
-      ],
+      ].filter(Boolean),
```
Comment (samrichca) [inline components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx:993]: that's a neat trick

File: components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx [hunk 4]
```diff
@@ -127,10 +127,14 @@ const DonorSummaryTable = ({
 
   const StatusColumnCell = ({ original }: { original: DonorSummaryRecord }) => (
     <CellContentCenter>
-      <StarIcon
-        fill={RELEASED_STATE_FILL_COLOURS[original.releaseStatus]}
-        outline={RELEASED_STATE_STROKE_COLOURS[original.releaseStatus]}
-      />
+      {original.validWithCurrentDictionary || FEATURE_SUBMITTED_DATA_ENABLED ? (
```
Comment (ciaranschutte) [inline components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx:130]: I don't understand the logic of using a feature flag or ( || ) the condition of `validWithCurrentDictionary`. 
If it's feature flagged surely we want that boolean by itself to control the logic here.

**Review by ciaranschutte — APPROVED** [review]

File: components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx (hunk 4 above)
Comment (samrichca) [reply components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx:130]: discussed on slack

Comment (ciaranschutte) [reply components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx:130]: "Well what's happening here is when Submitted Data is enabled, only the star is going to show, we're no longer showing the error Icon. So if the record is valid we show a star, or if Submitted Data is enabled we always show the star."

**Pull request merged**